recipe/meta.yaml

```diff
@@ -1,42 +1,64 @@
 {% set name = "slicer" %}
-{% set version = "0.0.7" %}
-
+{% set version = "0.0.8" %}
 
 package:
   name: {{ name|lower }}
   version: {{ version }}
 
 source:
   url: https://pypi.io/packages/source/{{ name[0] }}/{{ name }}/slicer-{{ version }}.tar.gz
-  sha256: f5d5f7b45f98d155b9c0ba6554fa9770c6b26d5793a3e77a1030fb56910ebeec
+  sha256: 2e7553af73f0c0c2d355f4afcc3ecf97c6f2156fcf4593955c3f56cf6c4d6eb7
+  patches:
+    - patches/0001-fix-tests-relative-imports.patch
```

Comment:
Did adding things like `slicer.utils_testing` to the imports cause the tests to fail? I'm curious why adding a patch instead of listing the necessary imports was required here?

Reply:
@Jrice1317  The tests fail because they are trying to import something from the same folder that is not there. They use relative imports (they start with a dot).

For example the line

```
from . import Slicer as S
```

means: from the current module, import `Slicer`. The test file (in this case there is only one test file) is isolated and alone in the tests folder and it should and cannot reference directly the source code, rather, the module in the package that we want to test: `slicer`.

For this reason, it is a general good practice to avoid relative imports in Python.

In the patch I replace relative imports like e.g.
```
from . import Slicer as S
```

to an absolute import, relative to the package we want to test `slicer`:
```
from slicer import Slicer as S
```

Reply:
The patch is not related to the `test/imports` section. It is relative to the test file `test_slicer.py` that is failing to import items because it tries to do it by using relative imports, and this file is isolated in a test folder for us (it would work if the source files and test file would be in the same folder as in the upstream repository).